packages/@sanity/form-builder/src/sanity/DefaultAssetSource/utils/mimeType.ts

```diff
@@ -0,0 +1,132 @@
+interface MimeTypeInfo {
+  title: string
+}
+
+const MIME_TYPES: Record<string, MimeTypeInfo> = {
```

Comment:
Some of these have suffixes ("Archive", "Audio") while others just have the file extension - would it make sense to hint at the type for all of them? `JPEG Image`, `AVI Video` etc?

Reply:
Discussed this with @jorngeorg, and we landed on it being smart to include it for all the files, so non-technical users easily can distinguish the type.